src/lhttpc_lb.erl

```diff
@@ -21,6 +24,7 @@
                 free=[] :: list()}).
 
 -export_types([host/0, port/0, max_connections/0, connection_timeout/0]).
+-type proplist() :: [{atom(),any()}].
```

Comment:
A proper type for a proplist is more like `[Key::any() | {Key::any(), Value::term()}]`.